trieste/models/gpflux/models.py

```diff
@@ -77,14 +75,12 @@ def __init__(
 
         self.original_lr = self.optimizer.optimizer.lr.numpy()
 
-        if not self.optimizer.minimize_args:
-            self._fit_args: Dict[str, Any] = {
+        if not self.optimizer.fit_args:
+            self.optimizer.fit_args = {
```

Comment:
Would be good to have a LR scheduler here? I'm actually unsure what optimizer is used by default, is it adam? I think adam + ReduceOnPlateau should be the default

Reply:
its adam, re ReduceOnPlateau I would wait for @sebastianober opinion